Source/init.cpp

```diff
@@ -139,20 +139,29 @@ void init_archives()
 {
 	std::vector<std::string> paths;
 	paths.reserve(5);
+#ifdef __ANDROID__
+	paths.push_back(std::string(getenv("EXTERNAL_STORAGE")) + "/");

```

Comment:
Would it perhaps be better to place the data files in a `devilutionx` subfolder or something?

Reply:
> We already `add_subdirectory` with SDL2 code, shouldn't we use the files directly from there instead of copying them here?

Probably, but my Java/Android skills aren't that strong.

Reply:
So you're saying we should defer changing it until we can settle on a well-defined pattern?

Reply:
That's my current thinking. But I will give it a quick look before merging the PR.

Reply:
Looks like the app adds `/devilutionx/` to the path, I think that is reasonable so I have added that for now. If there are any issues with this we should still have time to adjust :)